platform_registry_api/upstream_client.py

```diff
@@ -128,7 +128,10 @@ async def v2(self) -> dict[str, str]:
             return await response.json()
 
     async def list_images(
-        self, org: str, project: str, n: int | None = None, last: str | None = None
+        self,
+        org_project_filters: Sequence[str],
```

Comment:
if org_project_filters is empty the result will always be empty, right? no need to send request